homeassistant/components/smarty/sensor.py

```diff
@@ -0,0 +1,237 @@
+"""
+Support for Salda Smarty XP/XV Ventilation Unit Sensors.
+
+For more details about this component, please refer to the documentation at:
+https://home-assistant.io/components/sensor.smarty/
+"""
+import logging
+
+from homeassistant.core import callback
+from homeassistant.components.smarty import (
+    DATA_SMARTY, SMARTY_NAME, SIGNAL_UPDATE_SMARTY)
+from homeassistant.const import (
+    TEMP_CELSIUS, DEVICE_CLASS_TEMPERATURE,
+    STATE_ON, STATE_OFF, STATE_UNKNOWN)
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import Entity
+
+DEPENDENCIES = ['smarty']
+
+_LOGGER = logging.getLogger(__name__)
+
+DEVICE_CLASS_RPM = 'rpm'
```

Comment:
We can't invite custom device classes. Only device classes that are in the sensor base component are allowed.

Reply:
How should I do it instead? Should I ask or add it to homeassistant/const.py ?
I've seen some components doing this.

Reply:
No, just use the existing ones that fit the use cases here. For the remaining cases just `return None` for `device_class`.

New one device classes require approval in an issue in our architecture repo. That would take time.